dapp/src/utils/network.js

```diff
@@ -1,18 +1,19 @@
 import { networkIdToName } from 'constants/network'
+import invert from 'lodash/invert'
 
 export const isNetworkSupported = (networkType) => CONFIG.web3.supportedNetworks.includes(networkType)
 
 const blockExplorers = {
   main: 'https://etherscan.io',
   ropsten: 'https://ropsten.etherscan.io',
-  fuse: 'https://explorer.fuse.io'
+  fuse: 'https://explorer.fusenet.io'
 }
 
 export const getBlockExplorerUrl = (networkType) => blockExplorers[networkType]
 
 export const getApiRoot = (networkType) => CONFIG.api.url[networkType]
 
-export const isFuse = (provider) => provider.connection.networkVersion === '121'
+export const isFuse = (provider) => provider.connection.networkVersion === String(invert(networkIdToName)['fuse'])
```

Comment:
that kinda comlicate. let's just add the chainId to config file. then we'll use the config here. and define `networkIdToName` as:
networkIdToName = 
{
 ....
 [CONFIG...fuse]: fuse
 ...
}